rafx-shader-processor/src/lib.rs

```diff
@@ -533,32 +533,40 @@ fn process_glsl_shader(
     // Write out the spv and rust files if desired
     //
     if let Some(spv_file) = &spv_file {
-        std::fs::write(spv_file, output_spv)?;
+        write_output_file(spv_file, output_spv)?;
```

Comment:
This commit doesn't need to be included in this PR.

Reply:
I'll squash and merge in order so that they appear as separate changes.